ircd/resources.yaml

```diff
@@ -65,13 +70,55 @@ spec:
   - name: irc
     port: 6697
     targetPort: 6697
-  - name: websockets
+  - name: http
+    port: 80
+    targetPort: 80
+  - name: https
     port: 443
     targetPort: 443
   selector:
     app: irc
 ---
 apiVersion: v1
+kind: Service
+metadata:
+  name: webirc
+  labels:
+    app: webirc
+spec:
+  ports:
+  - port: 80
+    targetPort: 443
+  selector:
+    app: webirc
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: webirc-ingress
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    cert-manager.io/cluster-issuer: "letsencrypt-prod"
+    external-dns.alpha.kubernetes.io/hostname: irc.hashbang.sh
+spec:
+  tls:
+  - hosts:
+    - irc.hashbang.sh
+    secretName: webirc-tls
+  rules:
+  - host: irc.hashbang.sh
+    http:
+      paths:
+      - path: /*
+        backend:
+          serviceName: webirc
+          servicePort: 80
+      - path: /webirc
```

Comment:
funny that /webirc points at not webirc :P
Perhaps `/websocket` or something like that would be better?